README.md

```diff
@@ -20,6 +20,16 @@ pipenv run python setup_env.py
 The last command will install the git hooks that will run code formatting
 and linting on each commit.
 
+### Premainnet setup
+
+Premainnet setup works identically to the testnet setup, but requires two
+environment variables to be present: `BLOCKCHAIN` and `LIQUIDITY_ACCOUNT_PRIVATE_KEY`.
+
+The value of `BLOCKCHAIN` must be `premainnet`; the value of
```

Comment:
probably better to call it "network"?

Reply:
You think so? But in the SDK it is referred as "chain", no?
If you prefer "network", I'll change it.
What do you think?

Reply:
blockchain seems fine then!